conf/messages.en

```diff
@@ -1135,6 +1135,16 @@ houseConsignment.index.departureMeansOfTransport.country.check.error.required =
 
 unloadingFindings.rowHeadings.houseConsignment.countryOfDestination = Country of destination
 
+houseConsignment.index.documents.referenceNumber.NormalMode.title = What is the document’s reference number?
+houseConsignment.index.documents.referenceNumber.CheckMode.title = What is the new document’s reference number in house consignment {0}?
+houseConsignment.index.documents.referenceNumber.NormalMode.heading = What is the document’s reference number?
+houseConsignment.index.documents.referenceNumber.CheckMode.heading = What is the new document’s reference number in house consignment {0}?
+houseConsignment.index.documents.referenceNumber.hint = This can be up to 70 characters long and include both letters and numbers.
+houseConsignment.index.documents.referenceNumber.NormalMode.error.required = Enter the document’s reference number
+houseConsignment.index.documents.referenceNumber.CheckMode.error.required = Enter the new document’s reference number in house consignment {0}
+houseConsignment.index.documents.referenceNumber.CheckMode.error.invalid = The reference number must only include letters a to z and numbers 0 to 9
+houseConsignment.index.documents.referenceNumber.CheckMode.error.length = The reference number must be 70 characters or less
+houseConsignment.index.documents.referenceNumber.CheckMode.error.duplicate = The reference number must be unique
```

Comment:
If these last 3 message keys aren't mode specific, you could remove the mode from the keys